Tryouts/Core/Abstractions/Modules/ProcessInfo.cs

```diff
@@ -0,0 +1,38 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+namespace MorganStanley.ComposeUI.Tryouts.Core.Abstractions.Modules;
+
+public class ProcessInfo
```

Comment:
We can discuss changing the UIHint to the MainWindowHandle instead of ProcessId, but in general it should be possible to get that via the ProcessId

Reply:
It might be possible to get it via processid, but it is much easier to get it from the Process object returned by the windows. In fact, I do not even need the process id, just the window handle.

Reply:
Changed the UIHint for windowed modules to the MainWindowHandle. It is ensured to have a main window when the event arrives.